src/modules/Thread/context/hooks/useGetChannel.ts

```diff
@@ -24,29 +23,27 @@ export default function useGetChannel({
 }: DynamicProps, {
   sdk,
   logger,
-  threadDispatcher,
 }: StaticProps): void {
+  const {
+    actions: {
+      getChannelStart,
+      getChannelSuccess,
+      getChannelFailure,
+    },
+  } = useThread();
+
   useEffect(() => {
     // validation check
     if (sdkInit && channelUrl && sdk?.groupChannel) {
-      threadDispatcher({
-        type: ThreadContextActionTypes.GET_CHANNEL_START,
-        payload: null,
-      });
+      getChannelStart();
       sdk.groupChannel.getChannel?.(channelUrl)
         .then((groupChannel) => {
           logger.info('Thread | useInitialize: Get channel succeeded', groupChannel);
-          threadDispatcher({
-            type: ThreadContextActionTypes.GET_CHANNEL_SUCCESS,
-            payload: { groupChannel },
-          });
+          getChannelSuccess(groupChannel);
         })
         .catch((error) => {
           logger.info('Thread | useInitialize: Get channel failed', error);
-          threadDispatcher({
-            type: ThreadContextActionTypes.GET_CHANNEL_FAILURE,
-            payload: error,
-          });
+          getChannelFailure();
         });
     }
   }, [message, sdkInit]);
```

Comment:
여기 `message`가 아니라 `channelUrl`을 주는게 더 나을것같은데 어떻게 생각하시나요?

Reply:
diff에는 보이지 않는데, 해당 파일의 50번 라인부터 아래의 주석이 달려있습니다.
```
  /**
   * We don't use channelUrl here,
   * because Thread must operate independently of the channel.
   */
```

위의 주석 때문에 기존의 dependecy list를 바꾸지 않았습니다.